data/map altera.txt

```diff
@@ -4292,56 +4292,56 @@ system Zephyrius
-	description `	The Drak targeted this world, destroying most of its industrial capabilities and deactivating the replicators with a specific frequency wave signal, leaving the planet almost uninhabited and in heavily battle-scarred. Even now, the population has not gotten anywhere near its pre-war state, though the organic population has multiplied. As a result of the depopulation, nature has started to take back the cityscape, resulting in noticeable green patches.`
+	description `	The Drak targeted this world, destroying most of its industrial capabilities and deactivating the replicators with a specific frequency wave signal, leaving the planet almost uninhabited and heavily battle-scarred. Even today, the size of the population is not anywhere near its pre-war state. As a result nature has started to take back the abandoned areas, resulting in noticeable green patches.`
 	spaceport `The ecumenopolis appears to have many spaceports, of which only a handful are actually being used. None of them are small, but the main spaceport is truly gargantuan, large landing pads filled with freighters and transports, surrounded by multiple small towers that appear to be the spaceport authorities. A handful of hangars from the Alteran shipyards are also not too far off. Most interesting of all however is the outfitter selling replicators for use as temporary ship repair, an acceptable purpose even in the eyes of the Drak, as there does not seem to have been a second war.`
 	shipyard "Ancient Basic"
 	outfitter "Ancient Advanced"
 
 planet Atlantis
 	landscape land/beach11-harro
-	description `Floating in the middle of the ocean is the city of Atlantis, an Alteran city ship, you are unsure whether it is still capable of flying, but it may very well be. Apart from the luxury oriented shipyard and outfitter, there isn't much manufacturing going on here, mining and agriculture can only be found on the few small landmasses on this world and even then, it is barely present.`
+	description `Floating in the middle of the ocean is the city of Atlantis, an Alteran city ship. You are unsure whether it is still capable of flying, but it may very well be. Apart from the luxury oriented shipyard and outfitter, there isn't much manufacturing going on here, mining and agriculture can only be found on the few small landmasses of this world and even there, they are barely present.`
 	description `	Considering everything, there is no question as to why this is called a paradise world, the city is highly limited in terms of living space, due to its nature as a city ship, and is most definitely shielded and capable of submerging, a defensive system used during the Drak-Alteran War, explaining the myth of the sunken city of Atlantis. It's one of the safest places in Alteran space, if you can afford the exorbitant housing prices.`
```

Comment:
Just grammar corrections. A native english speaker should definitely review this again. :-)